### PR TITLE
Added CouchCMS

### PR DESCRIPTION
**Pull request by Ketrel**
Added CouchCMS to Blog and CMS sections

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,6 +134,7 @@ See also [Static Site Generators](#static-site-generators), [Content Management
   * [Antville](https://antville.org) - Antville is a free, open source project aimed at the development of a high performance, feature rich weblog hosting software. ([Source Code](https://github.com/antville/antville)) `Apache` `JavaScript`
   * [Blogotext](http://lehollandaisvolant.net/blogotext/en/) - A free blog-engine written in PHP and using SQLite. This offers you both an unmatched simplicity during installation and great performances. ([Source Code](https://github.com/timovn/blogotext)) `MIT` `PHP`
   * [Chyrp](https://github.com/chyrp/chyrp) - An extra-awesome, extra-lightweight blog engine. `MIT` `PHP`
+  * [CouchCMS](http://www.couchcms.com/) - Simple Open-Source CMS for designers ([Source Code](https://github.com/CouchCMS/CouchCMS)) `Common Public Attribution License 1.0` `PHP`
```
Comment (nodiscc) [inline README.md]: * CPALv1
 * Missing `.` after `designers`

File: README.md [hunk 2]
```diff
@@ -365,6 +366,7 @@ CMS are a practical way to setup a website with many features. CMS often come wi
   * [Bolt](https://bolt.cm/) - Sophisticated, lightweight & simple CMS. ([Source Code](https://github.com/bolt/bolt)) `MIT` `PHP`
   * [CMS Made Simple](http://www.cmsmadesimple.org/) - Open source content management system, faster and easier management of website contents, scalable for small businesses to large corporations. ([Source Code](http://svn.cmsmadesimple.org/svn/cmsmadesimple/trunk/)) `GPL` `PHP`
   * [Concrete 5 CMS](http://www.concrete5.org/) - An open source content management system. ([Source Code](https://github.com/concrete5/concrete5)) `MIT` `PHP`
+  * [CouchCMS](http://www.couchcms.com/) - Simple Open-Source CMS for designers ([Source Code](https://github.com/CouchCMS/CouchCMS)) `Common Public Attribution License 1.0` `PHP`
```
Comment (nodiscc) [inline README.md]: * License should be `CPALv1`
 * Missing `.` after `designers`

**Comment by Ketrel** [comment]
Corrected

**Comment by nodiscc** [comment]
May CMSs can be used as blogs. PLease remove CouchCMS from the blogs sections, CMS is enough.

**Pull request closed without merging**